### PR TITLE
Update documentation files

### PR DESCRIPTION
**Pull request by bs-sili**
## Description
Updating the documentation files in respect to the guidelines

### CODE REVIEW

File: .tractusx [hunk 1]
```diff
@@ -0,0 +1,6 @@
+product: "sldt-discovery-finder"
+leadingRepository: "sldt-discovery-finder"
+repositories:
+  - name: "sldt-discovery-finder"
+    usage: "This repository contains the source code for the SLDT Discovery Finder"
+    url: "https://github.com/eclipse-tractusx/sldt-discovery-finder.git"
```
Comment (tunacicek) [inline .tractusx:6]: empty line is missing

File: README.md [hunk 2]
```diff
@@ -1,24 +1,7 @@
-<!--
-    Copyright (c) 2021-2022 Contributors to the Eclipse Foundation
-
-    See the NOTICE file(s) distributed with this work for additional
```
Comment (tunacicek) [inline README.md:4]: Why you remove the header?

File: .tractusx (hunk 1 above)
Comment (bs-sili) [reply .tractusx]: empty line added.

Comment (bs-sili) [reply .tractusx]: And tractusx file updated

**Review by tunacicek — APPROVED** [review]
LGTM

**Pull request merged**